### PR TITLE
Add token

### PR DESCRIPTION
**Pull request by colin-msphere**
Add the ability to specify an authorization token when interacting with a DCOS cluster.

### CODE REVIEW

**Comment by ssk2** [comment]
Let's update the README too :)

**Comment by ssk2** [comment]
(Is there a way to extract this automatically from the DCOS CLI?)

**Comment by ianpward** [comment]
@ssk2 yes here is an example command: `curl -v -H "Authorization: token=$(dcos config show core.dcos_acs_token)" http://<dcos_url>/service/cassandra/v1/plan/`

Just to confirm this PR is for all API calls to Jenkins; not just the Marathon plugin?

**Comment by ssk2** [comment]
@ianpward - this PR is for running the demo script against a cluster with authentication. The Marathon plugin is being updated separately.

**Comment by ssk2** [comment]
@colin-msphere - could you have a look at my latest commit and check it? This now uses the local CLI configuration to fetch the token automatically (and errors if it's not set, or if it's not valid).

File: README.md [hunk 1]
```diff
@@ -32,7 +32,12 @@ When run with the `uninstall` command, it will:
     git clone https://github.com/mesosphere/cd-demo.git
     ```
 2. [Set up the DCOS CLI](https://docs.mesosphere.com/administration/introcli/cli/) locally.
-3. Ensure you have a DCOS cluster available. 1 node will work but more than 1 node is preferable to demonstrate build parallelism.
+3. Ensure you have a DCOS cluster available. 1 node will work but more than 1 node is preferable to demonstrate build parallelism. If you already had the CLI installed, make you sure you set the new cluster URL and authenticate against it:
```
Comment (colin-msphere) [inline README.md]: `make you sure`

**Comment by colin-msphere** [comment]
👍

**Pull request merged**